iphone/Classes/TiFilesystemFileProxy.m

```diff
@@ -153,6 +153,37 @@ -(id)spaceAvailable:(id)args
 	return [resultDict objectForKey:NSFileSystemFreeSize];
 }
 
+-(id)getProtectionKey:(id)args
+{
+	NSError *error = nil;
+	NSDictionary * resultDict = [fm attributesOfItemAtPath:path error:&error];
+	if (error!=nil) return NUMBOOL(NO);
+	NSString *obj = [resultDict objectForKey:NSFileProtectionKey];
+	if (obj != nil) {
+		if ([obj isEqualToString:NSFileProtectionNone]) {
+			return @"IOS_FILE_PROTECTION_NONE";
+		}
+		if ([obj isEqualToString:NSFileProtectionComplete]) {
+			return @"IOS_FILE_PROTECTION_COMPLETE";
+		}
+		if ([obj isEqualToString:NSFileProtectionCompleteUnlessOpen]) {
+			return @"IOS_FILE_PROTECTION_COMPLETE_UNLESS_OPEN";
+		}
+		if ([obj isEqualToString:NSFileProtectionCompleteUntilFirstUserAuthentication]) {
+			return @"IOS_FILE_PROTECTION_COMPLETE_UNTIL_FIRST_USER_AUTHENTICATION";
+		}
+	}
+}
+
+-(id)setProtectionKey:(id)args
```

Comment:
If the return type is know (in this case `NSNumber*`), declare the function as such:
`-(NSNumber*)setProtectionKey:(id)args`